internal/provider/provider.go

```diff
@@ -24,6 +24,7 @@ type provider struct {
 
 func (p *provider) GetSchema(_ context.Context) (tfsdk.Schema, diag.Diagnostics) {
 	return tfsdk.Schema{
+		Description: "A Provider for managing a Salesforce Organization",
```

Comment:
I didn't see this updated in the docs? Maybe it doesn't need to be, but wanted to point it out in case `generate` needs to be run, or it needs to be updated in a non-generated file. Or maybe nothing at all - just pointing it out :)

Reply:
It's possible the "updated doc generator" for the new protocol didn't cover this correctly, I'll double check and run make generate but my guess is it just isn't being generated